packages/teleport/src/services/clusters/types.ts

```diff
@@ -14,11 +14,13 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+type StatusEnum = 'online' | 'offline';
```

Comment:
`StatusEnum -> Status`

`Enum` has a special meaning in typesscript. The fact you used this suffix to name a type is confusing. 
In Javascript we used `Enum` suffix as a convention to indicate enumerations, no need to bring this convention to TS as TS has a built-in support for it.